examples/rl/deep_q_network_cartpole.py

```diff
@@ -0,0 +1,211 @@
+"""
+Title: Deep Q Network Method
+Author: [Jacob Chapman](https://twitter.com/jacoblchapman)
+Date created: 2020/05/23
+Last modified: 2020/05/23
+Description: Implement Deep Q Network in CartPole environment.
+"""
+"""
+## Introduction
+
+This script shows an implementation of Deep Q Network method on CartPole-V0 environment.
+
+### Deep Q Network Method
+
+As an agent takes actions and moves through an environment, it learns to map the observed
+state of the environment to an action. A Q-Learning Agent learns to perform its tasks,
+such that the recommended action maximizes the potential rewards.
+
+### CartPole-V0
```

Comment:
Since we already have a CartPole example, could we use another Gym environment for this one? One of greater complexity -- perhaps the bipedal walker, or an Atari game? Atari would be especially relevant given the history of deep Q-learning.

Reply:
Great Idea, I will try breakout and follow the deep mind paper. Only problem is the training time :)